charts/tesk/scripts/configure

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Is this file really needed? As we don't have any Jinja templates used with the Helm charts

Reply:
Left behind from previous deployment, good catch. I will delete it.